src/serlo-editor-repo/plugin-text/components/text-editor.tsx

```diff
@@ -157,7 +153,7 @@ export function TextEditor(props: TextEditorProps) {
         ({ value }) => ({ value, selection: previousSelection.current })
       )
     }
-    setHasSelectionChanged((selection) => selection + 1)
+    setSelectionChanged((selection) => selection + 1)
```

Comment:
Pls don't change this 🙏 

`selectionChanged` sounds like an event, like `mouseClicked` or `stateUpdated`

`hasSelectionChanged` sounds like a boolean, like `isButtonRed`. If you really dislike `has-` prefix, I'm ok with `isSelectionChanged`, but I'd really like to keep it obvious that this is a boolean flag.

Reply:
okay, glanced at some blog posts and I'm convinced :)
`isSelectionChanged` it is then.